doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -20,20 +20,148 @@
+| `req.inf.com.02` | Compute | The Architecture **should** include industry standard hardware management systems at both HW device level (embedded) and HW platform level (external to device). |
+| `req.inf.com.03` | Network | The Architecture **must** support Conatiner Runtime Interface (CRI). |
+| `req.inf.stg.01` | Storage | The Architecture **must**  support for Kubernetes Volumes for container storage.
+| `req.inf.stg.02` | Storage | The Architecture **may** provide shared Object storage as a service for Containers workload.
+| `req.inf.stg.03` | Storage | The Architecture **must** support Container Storage Interfaces (CSI).
+| `req.inf.stg.04` | Storage | The Architecture **may** support Software Defined Storage (SDS) that seamlessly supports shared block storage, object storage and flat files. |
+| `req.inf.stg.05` | Storage | The Architecture **should** provide high-performance and horizontally scalable storage. |
+| `req.inf.ntw.01` | Network | The Architecture **must** support intra-pod communications such as between all containers within a pod on localhost |
+| `req.inf.ntw.02` | Network | The Architecture **must** support intra-node communications, such as between agents on a node and all pods on that node |
+| `req.inf.ntw.03` | Network | The Architecture **must** support inter-node communications without NAT, such as communications between pods on a node with all other pods on all nodes |
+| `req.inf.ntw.04` | Network | The Architecture **must** support low latency and high throughput traffic needs. |
+| `req.inf.ntw.05` | Network | The Architecture **should** support service function chaining. |
+| `req.inf.ntw.06` | Network | The Architecture **must** support network resiliency. |
+| `req.inf.ntw.06` | Network | The Architecture **should** embrace the concepts of open networking and disaggregation using commodity networking hardware and disaggregated Network Operating Systems. |
+| `req.inf.ntw.07` | Network | The Architecture **should** embrace open-based standards and technologies. |
+| `req.inf.ntw.08` | Network | The Architecture **must** be capable of supporting highly available (Five 9’s or better) workloads. |
+| `req.inf.ntw.09` | Network | The Architecture **should** be architected to provide a standardised, scalable, and repeatable deployment model across all applicable NFVI sites. |
+| `req.inf.ntw.10` | Network | The networking solution **should** be configurable in an automated manner using openly published API definitions. |
+| `req.inf.ntw.11` | Network | The networking solution **should** be able to be centrally administrated and configured. |
+| `req.inf.ntw.12` | Network | The Architecture **must** support dual stack IPv4 and IPv6 for Kubernetes workloads. |
+| `req.inf.ntw.13` | Network | The Architecture **should** support IPv6 for Kubernetes workloads. |
+| `req.inf.ntw.14` | Network | The Architecture **must** support Conatiner Network Interface (CNI). |
```

Comment:
Can we update it as "The Architecture **must** support *co-exisence of multiple* Conatiner Network Interface (CNI)" ?

Reply:
@msidana That would require specific plugins -- adding a "should" requirement:
| `req.inf.ntw.16` | Network | The Architecture **should**  support co-exisence of multiple Conatiner Network Interface (CNI). |